README.md

```diff
@@ -291,6 +291,116 @@ do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com)
 which is generated at build-time on the server. Learn about rendering
 [`react` on the server][react-ssr] and [`glamor` too][glamor-ssr].
 
+### Example Style Objects
+
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
+selectors, introduce keyframe animations, and use media queries:
+
+<details>
+  <summary>pseudo-class</summary>
+  <p>
```

Comment:
Sorry about this. The ST package I'm using to preview the md doesn't handle details quite the same as GitHub or something. Let me know if the newest changes fix it.